books.rb

```diff
@@ -0,0 +1,10 @@
+class Books
+  attr_accessor :book, :author
+  attr_reader :rented
+
+  def initialize(book, author)
+    @book = book
+    @author = author
+    @rented = []
+  end
```

Comment:
- [ ] As it is a  requirement for this project, please a method for adding rentals, `the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` Checkout this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) and kindly fix that.